federatedscope/gfl/trainer/mamltrainer.py

```diff
@@ -1,48 +1,68 @@
 from federatedscope.register import register_trainer
 from federatedscope.core.trainers.trainer import GeneralTorchTrainer
 
+import learn2learn as l2l
```

Comment:
Is the new env package in the requirment.txt or the dockerfile?

Reply:
Currently the maml-based meta solution is only implemented in the contest branch. We'll discuss if it should be added into the master branch.